pymc4/utils.py

```diff
@@ -55,18 +56,34 @@ def merge_dicts(*dicts: dict, **kwargs: dict):
     return kwargs
 
 
-def biwrap(wrapper):
+def biwrap(wrapper) -> Callable:
+    """Allow for optional keyword arguments in lower level decoratrors.
+
+    Notes
+    -----
+    Currently this is only used to wrap pm.Model to capture model runtime flags such as
+    keep_auxiliary and keep_return. See pm.Model for all possible keyword parameters
+
+    """
+
     @functools.wraps(wrapper)
-    def enhanced(*args, **kwargs):
+    def enhanced(*args, **kwargs) -> Callable:
```

Comment:
@rpgoldman You've convinced me to start using type hints :). Did I do this right?

Reply:
That looks right. I believe that Callable can be specified further with arguments, though, if you know what arguments the returned function will take and/or you know what type it returns.
I'll have a peek now.

Reply:
Thanks for the quick answer, and adding type for function sounds fancy. I'll google around ot find examples.

Thanks again

Reply:
Are you missing an import for `Callable`?

Reply:
The import is here? 
https://github.com/pymc-devs/pymc4/pull/148/files#diff-03cbb4ef77afc157d399cf351453caf1R3